src/rpc/blockchain.cpp

```diff
@@ -689,11 +689,12 @@ UniValue getblockheader(const UniValue& params, bool fHelp)
 
 UniValue getblock(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2)
+    if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
-            "getblock \"hash\" ( verbose )\n"
+            "getblock \"hash\" ( verbose ) ( extraVerbose )\n"
```

Comment:
I do agree that for a new interface, a scale for verbosity would have made more sense instead of a boolean.